examples/gno.land/r/demo/postit/json.gno

```diff
@@ -0,0 +1,188 @@
+package postit
```

Comment:
Include a comment stating this file is temporary, pending JSON support implementation.